src/rust/ide/src/view/node_searcher.rs

```diff
@@ -84,28 +80,40 @@ impl NodeSearcher {
     /// Show NodeSearcher if it is invisible.
     pub fn show(&mut self) {
         if !self.is_shown() {
-            //FIXME:Use add_child(&text_field) when replaced by TextField 2.0
-            self.display_object.add_child(&self.text_field.display_object());
-            self.text_field.clear_content();
-            self.text_field.set_focus();
-            let graph      = self.node_editor.graph.controller().clone_ref();
-            let controller = controller::Searcher::new_from_graph_controller(&self.logger,&self.project,graph);
             let logger     = self.logger.clone_ref();
+            let position   = self.position();
+            let position   = position - self.node_editor.position();
```

Comment:
here I don't like that the same name `position` is used for two totally different positions…
do we need the former as a variable anyway, btw?